packages/typescript-resolver-files/src/generateResolverFiles/addExternalResolverImport.ts

```diff
@@ -10,6 +10,7 @@ interface AddExternalResolverImportParams {
    * 2. absolute import: ~<absolutePath>#{default as AliasIdentifier|named_import (as AliasIdentifier)}
    */
   configImportSyntax: string;
+  emitLegacyCommonJSImports: boolean;
```

Comment:
Could `emitLegacyCommonJSImports` come from `GenerateResolverFilesContext` like other functions in this PR :) ?